NeoDebuggerUI/ViewModels/MainWindowViewModel.cs

```diff
@@ -139,6 +148,16 @@ public void ClearLog()
             Log = "";
         }
 
+        public void AddBreakpoints()
+        {
+            //test only
```

Comment:
what do you mean by "test only"? Are you gonna remove this? If so, put a // TODO: <explain why you need to remove>
(TODO comments are easily found on the IDE)